src/2023/index.html

```diff
@@ -14,6 +14,9 @@ <h1 class="main-title">Helvetic Ruby 2023</h1>
           <p class="lead">A Ruby conference in Switzerland.</p>
           <p class="venue-date"><time>24.11.2023</time> in Bern, Switzerland</p>
 
+          <a href="mailto: keep-me-posted@helvetic-ruby.ch?subject=Keep%20me%20posted&body=[You%20can%20leave%20this%20part%20of%20the%20mail]" class="small shy-white-link">
+            Drop us your <em>email address</em> and you'll always be the first to receive the newest latest!
```

Comment:
The phrasing confused me. Would the following wording be okay as well?

```suggestion
            Drop us your <em>email address</em> and you'll always be the first to receive the latest news!
```